spec/feature/queue/judge_assignment_spec.rb

```diff
@@ -232,8 +232,9 @@
         click_on "Assign 2 cases"
         expect(page).to have_content("#{attorney_one.full_name} (0)")
         expect(page).to have_content("Error assigning tasks")
-        expect(page).to have_content("Docket (#{appeal_two.docket_number}) already "\
-                                     "has an open task type of #{JudgeDecisionReviewTask.name}")
+
+        expect(page).to have_content("Looks like this appeal already has an open JudgeDecisionReviewTask "\
+          "and this action cannot be completed.")
```

Comment:
Let's change this error message to:
```suggestion
        expect(page).to have_content("This appeal already has an open JudgeDecisionReviewTask "\
          "and this action cannot be completed.")
```

Reply:
The current error message was suggested by design in a slack conversation with @mbaenatan and @rutvigupta-design.  I'd appreciate their thoughts on any changes.